CRM/Member/Form/MembershipRenewal.php

```diff
@@ -610,22 +610,13 @@ protected function submit() {
 
     if (!empty($this->_params['record_contribution']) || $this->_mode) {
       // set the source
-      list($userName) = CRM_Contact_BAO_Contact_Location::getEmailDetails(CRM_Core_Session::singleton()->get('userID'));
+      [$userName] = CRM_Contact_BAO_Contact_Location::getEmailDetails(CRM_Core_Session::singleton()->get('userID'));
```

Comment:
@eileenmcnaughton is this the same?

Reply:
@seamuslee001 yep - my ide seems to keep doing it as the preferred new way